<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cyphers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Cypher">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4CAF50;  /* Задаем цвет фона */
    font: 11pt &quot;Segoe UI&quot;;
    color: white;  /* Задаем цвет текста */
    border: 1px solid rgb(154, 255, 143);  /* Задаем толщину и цвет границы */
    border-radius: 7px;  /* Закругляем углы */
    box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);  /* Добавляем тень */
    transition: background-color 0.3s, color 0.3s, box-shadow 0.3s, transform 0.3s;  /* Добавляем плавные переходы */
}

QPushButton:hover {
    background-color: #45a049;  /* Цвет фона при наведении */
    box-shadow: 0px 8px 16px rgba(0, 0, 0, 0.2);  /* Увеличиваем тень при наведении */
    transform: translateY(-2px);  /* Поднимаем кнопку на 2 пикселя */
}

QPushButton:pressed {
    background-color: #39803d;  /* Цвет фона при нажатии */
    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);  /* Уменьшаем тень при нажатии */
    transform: translateY(2px);  /* Поднимаем кнопку на 2 пикселя при нажатии */
}</string>
    </property>
    <property name="text">
     <string>Зашифровать</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-radius: 7px;  /* Закругляем углы */</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Stolb_Cypher">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Шифр:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Cypher">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-radius: 7px;  /* Закругляем углы */</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Decypher">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4CAF50;  /* Задаем цвет фона */
    font: 11pt &quot;Segoe UI&quot;;
    color: white;  /* Задаем цвет текста */
    border: 1px solid rgb(154, 255, 143);  /* Задаем толщину и цвет границы */
    border-radius: 7px;  /* Закругляем углы */
    box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);  /* Добавляем тень */
    transition: background-color 0.3s, color 0.3s, box-shadow 0.3s, transform 0.3s;  /* Добавляем плавные переходы */
}

QPushButton:hover {
    background-color: #45a049;  /* Цвет фона при наведении */
    box-shadow: 0px 8px 16px rgba(0, 0, 0, 0.2);  /* Увеличиваем тень при наведении */
    transform: translateY(-2px);  /* Поднимаем кнопку на 2 пикселя */
}

QPushButton:pressed {
    background-color: #39803d;  /* Цвет фона при нажатии */
    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);  /* Уменьшаем тень при нажатии */
    transform: translateY(2px);  /* Поднимаем кнопку на 2 пикселя при нажатии */
}</string>
    </property>
    <property name="text">
     <string>Расшифровать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Stolb_Decypher">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Расшифровка:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Cypher_f">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4CAF50;  /* Задаем цвет фона */
    font: 11pt &quot;Segoe UI&quot;;
    color: white;  /* Задаем цвет текста */
    border: 1px solid rgb(154, 255, 143);  /* Задаем толщину и цвет границы */
    border-radius: 7px;  /* Закругляем углы */
    box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);  /* Добавляем тень */
    transition: background-color 0.3s, color 0.3s, box-shadow 0.3s, transform 0.3s;  /* Добавляем плавные переходы */
}

QPushButton:hover {
    background-color: #45a049;  /* Цвет фона при наведении */
    box-shadow: 0px 8px 16px rgba(0, 0, 0, 0.2);  /* Увеличиваем тень при наведении */
    transform: translateY(-2px);  /* Поднимаем кнопку на 2 пикселя */
}

QPushButton:pressed {
    background-color: #39803d;  /* Цвет фона при нажатии */
    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);  /* Уменьшаем тень при нажатии */
    transform: translateY(2px);  /* Поднимаем кнопку на 2 пикселя при нажатии */
}</string>
    </property>
    <property name="text">
     <string>Зашифровать текст файла</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Decypher">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-radius: 7px;  /* Закругляем углы */</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Key">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>231</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-radius: 7px;  /* Закругляем углы */</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Stolb_Key">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>205</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Ключ:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Decypher_f">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4CAF50;  /* Задаем цвет фона */
    font: 11pt &quot;Segoe UI&quot;;
    color: white;  /* Задаем цвет текста */
    border: 1px solid rgb(154, 255, 143);  /* Задаем толщину и цвет границы */
    border-radius: 7px;  /* Закругляем углы */
    box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);  /* Добавляем тень */
    transition: background-color 0.3s, color 0.3s, box-shadow 0.3s, transform 0.3s;  /* Добавляем плавные переходы */
}

QPushButton:hover {
    background-color: #45a049;  /* Цвет фона при наведении */
    box-shadow: 0px 8px 16px rgba(0, 0, 0, 0.2);  /* Увеличиваем тень при наведении */
    transform: translateY(-2px);  /* Поднимаем кнопку на 2 пикселя */
}

QPushButton:pressed {
    background-color: #39803d;  /* Цвет фона при нажатии */
    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);  /* Уменьшаем тень при нажатии */
    transform: translateY(2px);  /* Поднимаем кнопку на 2 пикселя при нажатии */
}</string>
    </property>
    <property name="text">
     <string>Расшифровать текст файла</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>329</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_Shifr_Stolb">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Показывать только шифрованные символы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_All_Stolb">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Показывать все символы</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_Stolb_Key_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>45</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Исходный текст:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>337</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButton_Stolb_method">
       <property name="styleSheet">
        <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Улучшенный столбцовый метод</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Vizhioner_method">
       <property name="styleSheet">
        <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Метод Виженера(самоген. ключ)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButton_Save_reg">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>640</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Сохранять регистр</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
